superset/connectors/druid/models.py

```diff
@@ -1213,7 +1219,13 @@ def run_query(  # druid
 
         order_direction = "descending" if order_desc else "ascending"
 
-        if columns:
+        if (
+            IS_SIP_38
```

Comment:
Do you think you should add parenthesis to ensure the `or` applied to various blocks?